test/unit/connectionTest.ts

```diff
@@ -197,7 +226,9 @@ describe('Connection', () => {
   it('autoFetch() should reject the promise upon query error', async () => {
     const errorMsg = 'QueryFailed';
     requestMock.onSecondCall().throws(new Error(errorMsg));
-    const conn = await Connection.create({ authInfo: testAuthInfo as AuthInfo });
+    const conn = await Connection.create({
```

Comment:
Can these connection creates be safely moved to beforeEach?

Reply:
Their config (authInfo) varies a bit between tests, and would probably vary even more if we wanted to test more permutations.  So I'd say...probably not?

Reply:
the other option would be organizing them in another describe by authInfo, and putting a beforeEach to cover everything at that level?